### ISSUE TITLE
Replace DDG iframe with form

### ISSUE THREAD

**Issue by seanredmond** [seanredmond/Tindallgrams#29]
The contents of the iframe aren't accessible (no label from the input field, no accessible text for the button), so just replace it with a straight up form
